[2022-12-08 16:26:21,584] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_s3_to_redshift manual__2022-12-08T16:26:20.043649+00:00 [queued]>
[2022-12-08 16:26:21,590] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_s3_to_redshift manual__2022-12-08T16:26:20.043649+00:00 [queued]>
[2022-12-08 16:26:21,590] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:26:21,590] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-08 16:26:21,590] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-08 16:26:21,600] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): move_data_from_s3_to_redshift> on 2022-12-08 16:26:20.043649+00:00
[2022-12-08 16:26:21,605] {standard_task_runner.py:52} INFO - Started process 618 to run task
[2022-12-08 16:26:21,607] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'move_data_from_s3_to_redshidt_with_awswrangler', 'move_data_from_s3_to_redshift', 'manual__2022-12-08T16:26:20.043649+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/move_data_s3redshift.py', '--cfg-path', '/tmp/tmp50ofzf0w', '--error-file', '/tmp/tmpu2s9g56a']
[2022-12-08 16:26:21,608] {standard_task_runner.py:80} INFO - Job 99: Subtask move_data_from_s3_to_redshift
[2022-12-08 16:26:21,643] {task_command.py:369} INFO - Running <TaskInstance: move_data_from_s3_to_redshidt_with_awswrangler.move_data_from_s3_to_redshift manual__2022-12-08T16:26:20.043649+00:00 [running]> on host 8458a2c1ba2e
[2022-12-08 16:26:21,694] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=move_data_from_s3_to_redshidt_with_awswrangler
AIRFLOW_CTX_TASK_ID=move_data_from_s3_to_redshift
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T16:26:20.043649+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-08T16:26:20.043649+00:00
[2022-12-08 16:26:21,706] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:26:21,915] {credentials.py:1120} INFO - Found credentials in environment variables.
[2022-12-08 16:26:22,579] {python.py:173} INFO - Done. Returned value was: None
[2022-12-08 16:26:22,589] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=move_data_from_s3_to_redshidt_with_awswrangler, task_id=move_data_from_s3_to_redshift, execution_date=20221208T162620, start_date=20221208T162621, end_date=20221208T162622
[2022-12-08 16:26:22,620] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-08 16:26:22,647] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
